python/lsst/pipe/tasks/computeExposureSummaryStats.py

```diff
@@ -40,6 +40,50 @@
 
 class ComputeExposureSummaryStatsConfig(pexConfig.Config):
     """Config for ComputeExposureSummaryTask"""
+    doUpdatePsfModelStats = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Update the grid-based PSF model fidelity statistics (psfTraceRadiusDelta & psfApFluxDelta)?"
+        "Set to False if speed is of the essence.",
+    )
+    doUpdateApCorrModelStats = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Update the grid-based apCorr model fidelity statistic (psfApCorrSigmaScaledDelta)? "
+        "Set to False if speed is of the essence.",
+    )
+    doUpdateMaxDistToNearestPsfStats = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Update the grid-based maximun distance to the nearest PSF star fidelity statistic "
+        "(maxDistToNearestPsf)? Set to False if speed is of the essence.",
+    )
+    doUpdateWcsStats = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Update the wcs statistics? Set to False if speed is of the essence.",
+    )
+    doUpdatePhotoCalibStats = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Update the photoCalib statistics? Set to False if speed is of the essence.",
+    )
+    doUpdateBackgroundStats = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Update the background statistics? Set to False if speed is of the essence.",
+    )
+    doUpdateMaskedImageStats = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Update the masked image (i.e. skyNoise & meanVar) statistics? Set to False "
+        "if speed is of the essence.",
+    )
+    doUpdateEffectiveTimeStats = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Update the effective time statistics? Set to False if speed is of the essence.",
+    )
```

Comment:
I think you should remove these: the statistics computed on a grid should not be a performance hit, so there's no need to disable them.

Reply:
Huh, just want to double check that you really don't want control over any but the grid-based ones.  When we discussed this (way back when!), I was left with the strong feeling that you wanted control over almost all of them for AP (e.g. in the ticket description you mention `update_wcs_stats` and `update_masked_image_deltas` taking >1s per image as being a problem).  Does even if AP decides to keep many of them at present, does it hurt to have the option already available in case opinions on that change (for AP or other caller of this task)?

If you do want these removed, just for the record, can you specify on the ticket that you no longer consider any but the grid-based computations an issue?

Reply:
I've waffled back and forth on this a bit. The reason to not have `doX` configs for things that there would be no real reason to turn off is that it makes the code and config listings more complicated for no real benefit. I'll go add a comment on the ticket about this.

I've been doing some profiling but wanted to run on some OR4 data (which is more realistic than DC2) to get a better sense of the slow points.